src/common/CoreMediaIO/DeviceAbstractionLayer/Devices/Sample/Assistant/Server/Device/CMIO_DPA_Sample_Server_VCamDevice.cpp

```diff
@@ -32,10 +32,7 @@ namespace CMIO { namespace DPA { namespace Sample { namespace Server
         Device()
 	{
 		CreateStreams();
-        
-        obs_video_info ovi;
-        obs_get_video_info(&ovi);
-        mFrameSize = ovi.output_width * ovi.output_height * 2;
+        mFrameSize = getObsOutputWidth() * getObsOutputHeight() * 2;
```

Comment:
I wonder if `obs_get_video_info` is expensive to call. This calls it twice instead of once.

Reply:
your probably right, wether it is expensive to call or not, calling it only once should be better.